### PR TITLE
Created constants for ChannelServiceRoutes

### LINKED ISSUE #1475: [Skills] Turn ChannelServiceRoutes magic strings into constant(s)
1. Cleanup
2. Reusability for devs may not want the entire ChannelServiceRouter class
3. Though the requirement of proactive messaging renders HTTP as the default application layer protocol used, a WebSocket implementation of skills could provide a performance boost for parent-skill scenarios where proactive messages are not required

_Note: The current values used here may remain in the `botbuilder` library, as they are Node.js-specific. **`:conversationId`** in `/v3/conversations/:conversationId/activities` is the notation used in `express`, and `restify`._
___

[Example code](https://github.com/microsoft/botbuilder-js/blob/1f236f385faca6d3bee76f233ccefe43e5fc4856/libraries/botbuilder/src/channelServiceRoutes.ts#L20):

```ts
register(baseAddress, server) {
    server.post(baseAddress + '/v3/conversations/:conversationId/activities', this.processSendToConversation.bind(this));
    server.post(baseAddress + '/v3/conversations/:conversationId/activities/:activityId', this.processReplyToActivity.bind(this));
    server.put(baseAddress + '/v3/conversations/:conversationId/activities/:activityId', this.processUpdateActivity.bind(this));
    server.del(baseAddress + '/v3/conversations/:conversationId/activities/:activityId', this.processDeleteActivity.bind(this));
    server.get(baseAddress + '/v3/conversations/:conversationId/activities/:activityId/members', this.processGetActivityMembers.bind(this));
    server.post(baseAddress + '/v3/conversations', this.processCreateConversation.bind(this));
    server.get(baseAddress + '/v3/conversations', this.processGetConversations.bind(this));
    server.get(baseAddress + '/v3/conversations/:conversationId/members', this.processGetConversationMembers.bind(this));
    server.get(baseAddress + '/v3/conversations/:conversationId/pagedmembers', this.processGetConversationPagedMembers.bind(this));
    server.del(baseAddress + '/v3/conversations/:conversationId/members/:memberId', this.processDeleteConversationMember.bind(this));
    server.post(baseAddress + '/v3/conversations/:conversationId/activities/history', this.processSendConversationHistory.bind(this));
    server.post(baseAddress + '/v3/conversations/:conversationId/attachments', this.processUploadAttachment.bind(this));
}
```

### PR DESCRIPTION
**Pull request by mdrichardson**
Fixes #1475 

## Description

Provides constants used for ChannelServiceRoutes like:

> '/v3/conversations/:conversationId/activities'

## Testing

There are three tests that use these routes (`/v3/conversations/...`), but I did not make the tests use these constants. It made more sense to me have the tests use hard-coded strings, but let me know if I should change that.

### CODE REVIEW

File: libraries/botbuilder/src/routeConstants.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/* eslint-disable @typescript-eslint/no-namespace */
```
Comment (mdrichardson) [inline libraries/botbuilder/src/routeConstants.ts:8]: I followed the standard set in [AuthenticationConstants](https://github.com/microsoft/botbuilder-js/blob/master/libraries/botframework-connector/src/auth/authenticationConstants.ts), but [eslint didn't like that](https://github.com/typescript-eslint/typescript-eslint/blob/v1.13.0/packages/eslint-plugin/docs/rules/no-namespace.md). Let me know if I should change it up.

File: libraries/botbuilder/src/routeConstants.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    /**
+     * API path for single activity from conversation.
+     */
+    export const Activity = '/v3/conversations/:conversationId/activities/:activityId';
+
+    /**
+     * API path for all members from activity from conversation.
+     */
+    export const ActivityMembers = '/v3/conversations/:conversationId/activities/:activityId/members';
```
Comment (mdrichardson) [inline libraries/botbuilder/src/routeConstants.ts:60]: I considered doing something more like this:

image

...but it hurt readability. Let me know if you want me to go back to something more like the above, where the constants build on top of each other.

Comment (stevengum) [reply libraries/botbuilder/src/routeConstants.ts:60]: I like the constants you landed on; while it's not necessarily DRY it's just a bunch of strings, so simple is better..

File: libraries/botbuilder/src/routeConstants.ts (hunk 1 above)
Comment (stevengum) [reply libraries/botbuilder/src/routeConstants.ts:8]: Let's go the CallerIdConstants route in `botframework-schema` instead.

**Pull request merged**